src/app/shared/services/dashboard.service.ts

```
@@ -1,25 +1,25 @@
-  constructor(
-    private medicationsService: MedicationsService,
-    private seizuresService: SeizuresService,
-    private eventsService: EventsService,
-    private periodsService: PeriodsService
-  ) {}
+  private medicationsService: MedicationsService = inject(MedicationsService);
+  private seizuresService: SeizuresService = inject(SeizuresService);
+  private eventsService: EventsService = inject(EventsService);
+  private periodsService: PeriodsService = inject(PeriodsService);
+
+  constructor() {}
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)